boards.txt

```diff
@@ -948,3 +948,42 @@ DWM1001-DEV.menu.softdevice.s132.softdeviceversion=2.0.1
 DWM1001-DEV.menu.softdevice.s132.upload.maximum_size=409600
 DWM1001-DEV.menu.softdevice.s132.build.extra_flags=-DNRF52 -DS132 -DNRF51_S132
 DWM1001-DEV.menu.softdevice.s132.build.ldscript=armgcc_s132_nrf52832_xxaa.ld
+
+ArchLink.name=Seeed Arch Link
```

Comment:
Let's going with @jandrassy suggestion from https://github.com/sandeepmistry/arduino-nRF5/pull/311#issuecomment-441438774 and change the key to SeeedArchLink in boards.txt